karma.common.conf.js

```diff
@@ -139,28 +139,38 @@ let updatedConfig = null;
  * @param {object} webpackConfig Project webpack config
  * @param {boolean} typescriptProject true for typescript project
  */
-function updateDefaultConfig(projectName, karmaConfig, webpackConfig, typescriptProject=false) {
+function updateDefaultConfig(projectName, karmaConfig, webpackConfig, typescriptProject = false) {
   delete webpackConfig.entry;
   delete webpackConfig.output;
   webpackConfig.devtool = "inline-source-map";
   if (typescriptProject) {
     webpackConfig.module.rules[0].options = { configFile: "test/tsconfig.json" };
   }
+  updatedConfig = _.defaultsDeep(karmaConfig, defaultConfig);
 
-  webpackConfig.module.rules.push({
-    test: typescriptProject ? /\.tsx?$/ : /\.jsx?$/,
-    exclude: /node_modules/,
-    loader: "@jsdevtools/coverage-istanbul-loader",
-    enforce: "post",
-    options: {
-      esModules: true,
-    },
-  });
+  //special config for unit test debug
+  if (process.env.DEBUG_UNIT_TEST === "true") {
+    //since we're in debug, we do not want to run tests in headless
+    //we wait for a manual chrome visit to localhost:9876 to start tests
```

Comment:
```suggestion
    // since we're in debug, we do not want to run tests in headless
    // we wait for a manual chrome visit to localhost:9876 to start tests
```